<ui version="4.0" >
 <class>AddTask</class>
 <widget class="QWidget" name="AddTask" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Task</string>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>372</width>
     <height>399</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" >
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="font" >
           <font>
            <weight>50</weight>
            <bold>false</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text" >
           <string>Task Title</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>278</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="titleLine" />
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QCheckBox" name="dueDateCheck" >
          <property name="text" >
           <string>Due Date?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="annualCheck" >
          <property name="text" >
           <string>Annual Task?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>258</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QDateEdit" name="dateBox" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>238</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>288</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="description" />
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>188</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="cancelOk" >
        <property name="layoutDirection" >
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="standardButtons" >
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
